ui/page/wallets/wallet_page.go

```diff
@@ -1004,7 +1010,7 @@ func (pg *WalletPage) HandleUserInteractions() {
 											pg.Toast.Notify("Account created")
 											tim.Dismiss()
 										}
-										pg.loadWalletAndAccounts()
+										pg.UpdateAccountBalance()
```

Comment:
Does `UpdateAccountBalance` also update the accounts list? Because a new account has just been added.

Reply:
yes it does. This prevent the entire page from refreshing which in turn close all opened wallet modals when a new account is created.